project.md

```diff
@@ -58,11 +74,17 @@ making language design decisions), as opposed to "systems" aspects of the
 project (e.g., implementing a complicated semantics that doesn't require a lot
 of language design)?_
 
+_I think most of the time would be spent on the language aspects of this project because there is already a system set in place for javascript, which is what is currently used in this domain. I think figuring out how to implement the language in this environment would be complicated._
+
 ### Scope
 
 _How big an idea is this? How ambitious is this project?_
 
+_I think this is a really ambitious porject because its a language that would have to work with an already complicated piece of software._
```

Comment:
One possible simplification could be to create a DSL completely separate that outputs the JS to put into After Effects. I know this is a lot less satisfying since it would be less user-friendly, but whichever path you decide to go down, it could be a good place to start.